.eslintrc.js

```diff
@@ -0,0 +1,16 @@
+module.exports = {
+  'extends': 'airbnb',
```

Comment:
Lets add the react recommended rules: https://github.com/yannickcr/eslint-plugin-react